aws-android-sdk-kinesisvideo-archivedmedia/build.gradle

```diff
@@ -4,16 +4,12 @@ android {
     compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 15
+        minSdkVersion 21 // for kinesisvideo, which uses camera2 APIs.
```

Comment:
Hmm, will this have an impact on our existing customers? What was happening currently with the minSdkVersion being too low? Is there only a limited number of methods needing this we could wrap with RequiresAPI annotations?

Reply:
Previously, the `lintOptions { abortOnError false }` was suppressing the build error about the `minSdkVersion` being too low.

The API level is ultimately coming from a few files in the `-kinesisvideo` package, which then bump up this package. Let me see if I can use the `RequiresVersion`, to lower requirements on the two packages.

Reply:
I started going down this path -- it is ultimately a lot of files that raise that API level. I think it'd be best to do it globally on the module for now. If people complain and need lower API level support, we could re-evaluate in the future.